src/Model/ModelData.php

```diff
@@ -48,16 +45,18 @@ class ModelData
      * </code>
      */
     private static array $casting = [
-        'CSSClasses' => 'Varchar'
+        'CSSClasses' => 'Varchar',
+        'forTemplate' => 'HTMLText',
```

Comment:
`forTemplate()` returns a value that are ready to be _directly_ injected in a template completely raw. Usually it's HTML markup specifically.